Readability.js

```diff
@@ -624,10 +623,10 @@ Readability.prototype = {
         // Move all of the page's children into topCandidate
         topCandidate = doc.createElement("DIV");
         neededToCreateTopCandidate = true;
-        var children = page.childNodes;
-        while (children.length) {
-          this.log("Moving child out:", children[0]);
-          topCandidate.appendChild(children[0]);
+        var kids = page.childNodes;
```

Comment:
Why are we still using childNodes here instead of children? If this is correct, I think we should update the comment above to talk about nodes instead of "children".